### DISCUSSION TITLE
Chemistry based on Distilling, Stabilising, Breaking down chemicals and Reaction monitoring

### DISCUSSION THREAD

**Discussion by Bod9001** [unitystation/unitystation#8360] [Ideas]
# Preliminary information
A chemistry system based on Distilling, Reactions not 100% reacting (So they have to be distilled To get desired product), Reactions that have to be closely monitored so they don't fail (The more powerful the end result the harder it is to keep stable), Stabilising ingredients that reduce the power but makes it easier to make , Mixing of Chemicals and Good old dispenser
 
it Was Talked with Satou Tatsuhiro In the discord about this and it sounded like a good idea so Here it is!
## Category
Feature Request | Chemistry

Requested Behavior
so When starting to make a certain chemical

- You first get the parts you need (From dispenser(If it dispenses it), Broken down from other chemicals like water from the dispenser, if contraband distilled from certain station items)

- Put into heating and mixing Machine (That you can control speed of mixing and or temperature

- Putting in stabiliser (that can make the reaction easier to do but reducing the purity, If you want to make it more pure you can use less effective Stabilising agents or non)  

- Then add The other components of the chemical (If an unstable process then different mixing speeds and temperatures and  the rate which you put the other components in, Would have to be adjusted so the chemical won't react incorrectly giving you a Different chemical instead or An impure solution)

- Take the reacted solution then Distil to get the certain chemical you want ((Due to the reaction not Making 100% of what you want (will always do unless you do it 100% perfectly that would be **very** hard to do) and or Unwanted byproducts Being in the solution) You control it by setting the temperature and what temperatures to take liquids from Because every chemical has its own gas temperature)

- then slap it into a ChemMaster  And get your pills/patches/bottled solution

**Comment by moonheart08** [comment]
This would be neat, but at the same time you want to avoid making it too boring. I think the described long winded process would be a bit of a pain just to make some spaceacillin 50 times a round.

**Comment by Bod9001** [comment]
well I was thinking the processes would be very quick but you have to concentrate on them, try to reduce  the "you need to make this to make that", changing it to you can make something that makes it easier but you can do it with the base stuff you just have to be really careful.

Like the speed for distilling would be 1u used every second

**Comment by fomalsd** [comment]
isn't this kind of done?

**Comment by Bod9001** [comment]
humm no, We currently have chemistry system, This is proposing a more complex system where you have to monitor reactions and it takes time And you have to distil to get rid of impuritys

**Comment by qutoh** [comment]
Making a chemistry system closer to life is a hell of a rabbit-hole to go down - nothing _ever_ goes to 100% when reacting (even when you do everything perfectly) especially not complex organic drugs, because every chemical reaction is in a dynamic equilibrium reversing back to what made it. Where that equilibrium lies is different for each reaction, as is the speed. In general hot things react faster, also obviously liking to explode faster, and explodey things also tend to reach closer to 100% yield (are more reactive) but there are reactions where there's a kinetic product and thermodynamic product (more of one thing at a low temperature, more of the other at higher,) etc. etc. It's hellishly complicated.

Chemical engineers, especially in pharma, don't tolerate less than optimum. The main problem affecting most reactions' yield are completely unavoidable (but often valuable) side-products, not what's going on with your reaction vessel. They make a system for each reaction that exploits LeChatlier's Principle - you upset equilibrium in either direction, the reaction tends to go back to it, so drawing out your product continually will make more of the chemical whosits react. They design that system so that it's as hands-off as humanly possible, because trusting interns with millions of dollars of equipment is hairy enough without the risk that we might turn a dial wrong.

My question as a chemtech is, if you have magical chemdispensers that can pull reagents from storage instantly, miracle synthesis routes that have basically instantaneous kinetics which form no chiral side-products, stabilizing complexes that are suited to multiple explosives, and microfluidics to turn anything and everything into pills, patches, and bottles no matter how unstable they'd be in each... Why not also the basic stuff that gives good yield outside a lab?

**Comment by Bod9001** [comment]
The main idea of it was, instead of you pressing buttons. You just build a contraption to mix them for you, where the limitations are not the time, but the purity, it's kind of changed a bit since I first thought of it, but  that's kind of the gist,  difficulty can range from mixed 2 chemical together boom you got your chemical, you have to ensure there's a bit of catalyst there and it ends up in the end, and it requires some processes to remove it (gets easy with technology upgrades), and then the hardest would be where you have to monitor the reaction because it's very close to the explosion threshold, and there's quite a few catalysts that you have to remove, but it gives you a very powerful chemical, especially this can be all made into a production line of sorts with connecting different stuff to different stuff

**Comment by Bod9001** [comment]
Rock  (ore got from mining ( like the the excess ) you can search Through it for chemicals )
Plant  (botany growing plants and stuff with chemicals)
Fire (burning stuff on station like burning cables to get the fumes off it)
Bio  (viruses/bacteria  that generate what you want )
Water (base for some reactions Like hydrogen and oxygen)
Air ( Like reacting oxygen with your chemicals to get something )

idea for chemical scarcity

important note caveman with cave basic tools Should be able to do basic things 

Like burning something Then mixing the ashes With some water in a Of some sort and then letting it sit for a bit to react 
Making that healing chemical

Example of how I mean it to be, that with basic supplies you can make something that's actually terrible at healing but in a pinch is better than nothing

**Comment by PerfectTangent** [comment]
this is probably best served as a discussion.
